README.md

```diff
@@ -5,15 +5,13 @@ Processor SDK Documentation in Sphinx
 
 ### Install tools on Ubuntu
 
-    $ sudo apt-get install python-sphinx
-
-OR use the following command in a python virtual environment for a known working
-config as of 2023/09/05:
+Use the following command in a python virtual environment for a known working
+config:
 
     # python3 -m pip install -r requirements.txt
 
 OR you can use a docker container like the following:
-    - [psdk-doc-docker](https://github.com/StaticRocket/psdk-doc-docker/pkgs/container/psdk-doc-docker)
+    - [psdk-doc-docker](https://github.com/TexasInstruments/processor-sdk-doc/pkgs/container/processor-sdk-doc)
```

Comment:
why not link this to the readme.md file we are just adding?

Reply:
This links to the page that will be created on first run that shows all tags and the README in one place.

Reply:
I meant put that information in the docker/readme.md and point using :ref: to that doc